ballerina-tests/tests/50_bidi_caller_cancel_status_service.bal

```diff
@@ -23,18 +23,18 @@ boolean cancelled = false;
 service "HelloWorld50" on ep50 {
 
     remote function sendString(HelloWorld50StringCaller caller,
-     stream<string, error?> clientStream) returns error? {
-        record {|string value;|}|error? result = clientStream.next();
-        result = clientStream.next();
+    stream<string, error?> clientStream) {
+        _ = checkpanic clientStream.next();
+        _ = checkpanic clientStream.next();
         if caller.isCancelled() {
             cancelled = true;
         }
-        _ = check clientStream.next();
-        check caller->complete();
+        _ = checkpanic clientStream.next();
```

Comment:
```suggestion
        _ = check clientStream.next();
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/c32ef7e98e2bfaa4df966a96025d16d755dfb1cd